setup.py

```diff
@@ -302,7 +302,7 @@ def get_exts_for(name):
                         recursive=True,
                     )
                 ),
-                extra_compile_args=debug_compile_args + ["-std=c++17"],
+                extra_compile_args=extra_compile_args + debug_compile_args + ["-std=c++17"],
```

Comment:
@gnufede I think this might need to use the windows formatting of the flag `"/std:c++17"`